src/app/containers/CpsMetadata/utils.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line import/prefer-default-export
+export const getBrandedImage = (image, service) => {
+  // const imageUrl = image
+  //   .split('/')
+  //   .splice(3)
+  //   .join('/');
```

Comment:
Let's not forget to uncomment :)

Reply:
Sure, I'm not entirely sure this is the right way to get this done. Still waiting on a discussion with Chris to finish up this part.